src/main/webapp/scripts/private-room-script.js

```diff
@@ -50,3 +50,23 @@ function onPlayerStateChange(event) {
 function stopVideo() {
 	youtubePlayer.stopVideo();
 }
+
+async function displayChat() {
+    let response = await fetch('/chat');
+    let messages = await response.json();
+    const messageElement = document.getElementById('chat-messages');
+    messageElement.innerHTML = '';
+    for (message in messages) {
+      commentElement.appendChild(createParagraph(comments[comment]));
+    }  
+}
+
+function createParagraph(msgJson) {
+    var msg = JSON.parse(msgJson);
+    const paragraph = document.createElement('p');
+    paragraph.innerHTML = '<h4>';
+    paragraph.innerText = msg.sender;
+    paragraph.innerHTML = '</h4>';
+    paragraph.innerText = msg.message;
```

Comment:
paragraph.innerHTML = `<h4${msg.sender}</h4${msg.message}

It doesn't allow me to write the exact code. Please read carefully

Reply:
This function has been updated and renamed since then. Pushing new version